<launch>
  <arg name="sync" default="false" />
  <arg name="leaf_size" default="0.1" />
  <arg name="node_name" default="voxel_grid_filter" />
  <arg name="points_topic" default="points_raw" />
  <arg name="output_log" default="false" />
  <arg name="measurement_range" default="200" />

  <node pkg="points_downsampler" name="$(arg node_name)" type="$(arg node_name)" output="screen">
    <param name="input_topic_name" value="points_raw" />
    <param name="output_topic_name" value="filtered_points" />
    <param name="leaf_size" value="$(arg leaf_size)" />
    <param name="measurement_range" value="$(arg measurement_range)" />
    <param name="output_log" value="$(arg output_log)" />
  </node>
</launch>
